runtime/tests/pallet_proxy.rs

```diff
@@ -184,18 +184,18 @@ fn test_proxy_pallet() {
         call_register,
     ];
 
-    for call in calls.iter() {
-        for proxy_type in proxy_types.iter() {
-            new_test_ext().execute_with(|| {
-                assert_ok!(Proxy::add_proxy(
-                    RuntimeOrigin::signed(AccountId::from(ACCOUNT)),
-                    AccountId::from(DELEGATE).into(),
-                    *proxy_type,
-                    0
-                ));
-
-                verify_call_with_proxy_type(proxy_type, &call());
-            });
-        }
-    }
+    // for call in calls.iter() {
```

Comment:
Why is this commented out ? If is not needed anymore would it make sesne to delete it ?

Reply:
thanks for review. the PR is in progress, not ready for review yet. I set it as in progress, maybe I should set it as draft. but I am not sure if CI works for draft PR.